lib/chelsea/gems.rb

```diff
@@ -28,6 +33,12 @@ def initialize(file, options)
       )
     end
 
+    def get_db_store_location()
+      initial_path = File.join("#{Dir.home}", ".ossindex")
+      Dir.mkdir(initial_path) unless File.exists? initial_path
+      path = File.join(initial_path, "chelsea.pstore")
+    end
```

Comment:
Is using a dir in the current user's home standard practice for caching in our scan integrations? Seems important to surface this fact in the README.

Reply:
Yeah, that location is common amongst the rest of the integrations. I figured homedir as it is more than likely writeable by any user.